UPGRADE-6.0.md

```diff
@@ -23,7 +23,8 @@ Console
 -------
 
  * `Command::setHidden()` has a default value (`true`) for `$hidden` parameter
- * Remove `Helper::strlen()`, use `Helper::width()` instead.
+ * Removed `Helper::strlen()`, use `Helper::width()` instead.
+ * Removed `Helper::strlenWithoutDecoration()`, use `Helper::removeDecoration()` instead.
```

Comment:
This file should not be updated in the 6.0 branche

Reply:
fixed in #41313